### ISSUE TITLE
Repoint mapping from Mapserver to Geoserver

### ISSUE THREAD

**Issue by LynnHeeley** [jncc/marine-habitat-classification#69]
Currently the maps are pulled from Mapserver.  With the new MPA Mapper served by Geoserver these links will need to be updated. Marine responsible for creating the point layers in Geoserver. New MPA mappers live end May.

**Comment by CathyJinJNCC** [comment]
This hasn't been done yet

**Comment by jonparsonsjncc** [comment]
cathy to talk to greme

**Comment by CathyJinJNCC** [comment]
The new geoserver is currently using an mpa workspace and the plan is to switch it over to a uk mapping workspace which, from my understanding, requires some dev time (from Paul?) to set up more databases for the extra data sources. It doesn't make sense to migrate the MHC data over to the mpa workspace/database only for it to then be switched over.

**Comment by helen-lillis** [comment]
@CathyJinJNCC yes that's right, we're waiting on Paul (or someone he can delegate?) to set up the new database on PostgreSQL. From there we should be able to create the new UK mapping GeoServer workspace and reproduce the maps from there.

**Comment by jonparsonsjncc** [comment]
Hi @paul-gilbertson  just checking to see if this task has been done and we can close this ticket
Thanks

**Comment by helen-lillis** [comment]
@jonparsonsjncc this one still hasn't been done. We're still waiting for DDS input.

**Comment by jonparsonsjncc** [comment]
Hi @helen-lillis 
I have created a new task for this
https://app.zenhub.com/workspaces/dev-board-59b7d1ba3d598a066f875855/issues/jncc/dev/326
and am closing this old one
